src/app/shell/result/workshop-map-view-list/workshop-map-view-list.component.ts

```diff
@@ -30,8 +32,8 @@ import { RegistrationState } from 'src/app/shared/store/registration.state';
 })
 export class WorkshopMapViewListComponent implements OnInit, OnDestroy {
 
-  constructor(private store: Store) { }
   destroy$: Subject<boolean> = new Subject<boolean>();
+
   @Input() public filteredWorkshops$: Observable<WorkshopFilterCard>;
```

Comment:
@Select(FilterState.filteredWorkshops)
  filteredWorkshops$: Observable<WorkshopCard[]>;


typo is not the same as it is passed from parent-component.